views/quiz.hbs

```diff
@@ -42,7 +42,18 @@
           <div class="answer-container-contain" data-bhex="{{primary_hex}}" data-fhex="{{secondary_hex}}">
             <div class="answer-row subheader">Players Guessed</div>
 
-            {{#compare type 'leaders'}}
+            {{#compare type "goats"}}
```

Comment:
probably a shorter way to do this without repeating 10 lines of code.. but I couldn't figure out how